examples/react-native/v12/TestApp/src/components/quickstart/nonsync/RealmWrapper.test.tsx

```diff
@@ -0,0 +1,110 @@
+import 'react-native';
+import React from 'react';
+import {AppWrapper} from './RealmWrapper';
+import {render, screen, userEvent} from '@testing-library/react-native';
+
+const user = userEvent.setup();
+
+describe('Quick Start minimum viable app', () => {
+  beforeEach(async () => {
+    render(<AppWrapper />);
+
+    const createInput = await screen.findByTestId('create-input');
+    const createProfileButton = await screen.findByTestId('create-profile');
+
+    // Create test objects
+    await user.type(createInput, 'Book Worm');
+    await user.press(createProfileButton);
+    await user.clear(createInput);
+    await user.type(createInput, 'testProfile');
+    await user.press(createProfileButton);
+    await user.clear(createInput);
+  });
+
+  afterEach(async () => {
+    const removeButton = await screen.findByTestId('remove-all-objects');
+
+    // Delete any existing realm objects
+    await user.press(removeButton);
+  });
+
+  test('Read', async () => {
+    // Get profile lists
+    const profiles = await screen.findAllByTestId('profile');
+    expect(profiles.length).toBeGreaterThan(0);
+
+    const sortedProfiles = await screen.findAllByTestId('sorted-profile');
+    expect(sortedProfiles.length).toBeGreaterThan(0);
+    // First profile in list should be "Book Worm"
+    expect(sortedProfiles[0].children[1]).toBe('Book Worm');
+
+    const filteredProfiles = await screen.findAllByTestId('filtered-profile');
+    // Should only be one profile
+    expect(filteredProfiles.length).toEqual(1);
+    // Profile should match "testProfile"
+    expect(filteredProfiles[0].children[1]).toBe('testProfile');
+  });
+
+  test('Create', async () => {
+    const createInput = await screen.findByTestId('create-input');
+    const createProfileButton = await screen.findByTestId('create-profile');
+    const profiles = await screen.findAllByTestId('profile');
+    // Should already have 2 objects from `beforeEach()`
+    expect(profiles.length).toBeGreaterThan(0);
+
+    // Create test objects
+    await user.type(createInput, 'React Native');
+    await user.press(createProfileButton);
+    await user.clear(createInput);
+
+    const refreshedProfiles = await screen.findAllByTestId('profile');
+    // Rerendered list should have more profile items than the
+    // initially-rendered list
+    expect(refreshedProfiles.length).toBeGreaterThan(profiles.length);
+  });
+
+  test('Update', async () => {
+    const profilesToUpdate = await screen.findAllByTestId('profile-to-update');
+    // Should already have 2 objects from `beforeEach()`
+    expect(profilesToUpdate.length).toBeGreaterThan(0);
+
+    // Select first profile in list to update.
```

Comment:
```suggestion
    // Select first profile in list to update
```
[nit] for consistency